packages/nimble-components/docs/css-guidelines.md

```diff
@@ -126,6 +126,95 @@ For example:
 
 Useful reference: [When do the :hover, :focus, and :active pseudo-classes apply?](https://bitsofco.de/when-do-the-hover-focus-and-active-pseudo-classes-apply/)
 
+## Hierarchical Cascade Layers with @layer
```

Comment:
Some high-level thoughts:

- Since these are guidelines, the title should be a clear action
- I think the advice is a bit too hard to evaluate. I think we can simplify considerably and just say new control styles should be organized this way.
- This feels directly related to the previous topic and is describing implementation details for that topic, I think we can merge it there as a subheader.
- The example should be simplified and made standalone. I'd avoid referencing source code directly if it's not needed. Think folks can will try and search for examples in source if they want it, we don't need to maintain that link.

So at a high-level I think the docs should be organized like this (the following is outlining and should be word smithed):

```
### Organize styles with CSS Cascade Layers

New controls and existing controls undergoing major refactoring should migrate to using [CSS cascade layers](link to mdn) for organizing styles that override states.
Cascade layers help enforce that property overrides have higher specificity over the base styles without modifying selectors to increase specificity.

For consistency, control styles that adopt CSS Cascade Layers should follow these practices:
- Specify the following layers in: `@layer base, hover, focusVisible, active, disabled, top`
   - Avoid changing the layer order of adding additional layers.
- Ensure all styles are placed in a layer and no styles for the control are outside a layer.
- Ensure the layers in the stylesheet are ordered based on the layer order.
- Ensure the styles within a layer follow existing best practices such as grouping using document order.

Example:
` ` `css
@layer base, hover, focusVisible, active, disabled, top

@layer base {
    :host {
        border: green;
   }
}
@layer hover {}
@layer focusVisible {}
@layer active {}
@layer disabled {
    :host([disabled]) {
        border: gray;
   }
}
@layer top {}
` ` `
```